pkgs/by-name/su/superhtml/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  callPackage,
+  fetchFromGitHub,
+  stdenv,
+  zig,
+}:
+stdenv.mkDerivation rec {
+  pname = "superhtml";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "kristoff-it";
+    repo = "superhtml";
+    rev = "v${version}";
```

Comment:
`refs/tags/v${version}` would be preferred here.

Reply:
Fixed